test/e2e/set/remotekubernetescluster/remotekubernetescluster_finalizer.go

```diff
@@ -0,0 +1,136 @@
+		err = cluster.ScyllaAdminClient().ScyllaV1alpha1().RemoteKubernetesClusters().Delete(ctx, rkcName, metav1.DeleteOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Awaiting RemoteKubernetesCluster %q deletion", rkc.Name)
+		err = framework.WaitForObjectDeletion(ctx, f.DynamicAdminClient(), scyllav1alpha1.GroupVersion.WithResource("remotekubernetesclusters"), rkc.Namespace, rkc.Name, nil)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Recreating RemoteKubernetesCluster %q", rkcName)
+		rkc = originalRKC.DeepCopy()
+		rkc, err = cluster.ScyllaAdminClient().ScyllaV1alpha1().RemoteKubernetesClusters().Create(ctx, rkc, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the RemoteKubernetesCluster %q to rollout (RV=%s)", rkc.Name, rkc.ResourceVersion)
+		waitCtx2, waitCtx2Cancel := utils.ContextForRemoteKubernetesClusterRollout(ctx, rkc)
+		defer waitCtx2Cancel()
+
+		rkc, err = controllerhelpers.WaitForRemoteKubernetesClusterState(waitCtx2, cluster.ScyllaAdminClient().ScyllaV1alpha1().RemoteKubernetesClusters(), rkc.Name, controllerhelpers.WaitForStateOptions{},
+			utils.IsRemoteKubernetesClusterRolledOut,
+			hasRKCFinalizer,
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Creating ScyllaDBCluster using RemoteKubernetesCluster %q", rkcName)
+		sc1 := f.GetDefaultScyllaDBCluster([]*scyllav1alpha1.RemoteKubernetesCluster{rkc})
+
+		sc1, err = cluster.ScyllaAdminClient().ScyllaV1alpha1().ScyllaDBClusters(f.Namespace()).Create(ctx, sc1, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the ScyllaDBCluster %q to rollout (RV=%s)", sc1.Name, sc1.ResourceVersion)
+		waitCtx3, waitCtx3Cancel := utils.ContextForMultiDatacenterScyllaDBClusterRollout(ctx, sc1)
+		defer waitCtx3Cancel()
+		sc1, err = controllerhelpers.WaitForScyllaDBClusterState(waitCtx3, cluster.ScyllaAdminClient().ScyllaV1alpha1().ScyllaDBClusters(f.Namespace()), sc1.Name, controllerhelpers.WaitForStateOptions{},
+			utils.IsScyllaDBClusterRolledOut,
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Deleting RemoteKubernetesCluster %q", rkc.Name)
+		err = cluster.ScyllaAdminClient().ScyllaV1alpha1().RemoteKubernetesClusters().Delete(ctx, rkcName, metav1.DeleteOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		rkc, err = cluster.ScyllaAdminClient().ScyllaV1alpha1().RemoteKubernetesClusters().Get(ctx, rkcName, metav1.GetOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(rkc.DeletionTimestamp).NotTo(o.BeNil())
+
+		hasIsBeingUsedCondition := func(rkc *scyllav1alpha1.RemoteKubernetesCluster) (bool, error) {
+			cond := meta.FindStatusCondition(rkc.Status.Conditions, "RemoteKubernetesClusterFinalizerProgressing")
+			return cond != nil && cond.Reason == "RemoteKubernetesClusterFinalizerProgressing_IsBeingUsed", nil
+		}
+		framework.By("Awaiting is being used condition on RemoteKubernetesCluster %q", rkc.Name)
```

Comment:
```suggestion
		framework.By("Awaiting 'IsBeingUsed' condition on RemoteKubernetesCluster %q", rkc.Name)
```
nit, difficult to read now

Reply:
done